unified-installer/astra-unified-installer.sh

```diff
@@ -1851,11 +1851,20 @@ step_generate_astra_connector_yaml() {
 
     # Default memory limit
     memory_limit=2
-    snapshot_count=""
```

Comment:
Just noticed, these are global since they're not declared as local -- any chance we could change them to local as part of this PR?

Reply:
thanks for that, teach me the bash ways!